### PR TITLE
Converting mongodb deployment to statefulset

### PR DESCRIPTION
**Pull request by rajdas98**
Signed-off-by: rajdas98 <mail.rajdas@gmail.com>

## Proposed changes

Converting mongodb deployment to statefulset

## Types of changes

What types of changes does your code introduce to Litmus? Put an `x` in the boxes that apply
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Bugfix (non-breaking change which fixes an issue)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [ ] Documentation Update (if none of the other choices applies)

## Checklist

Put an `x` in the boxes that apply. You can also fill these out after creating the PR. If you're unsure about any of them, don't hesitate to ask. We're here to help! This is simply a reminder of what we are going to look for before merging your code.
- [ ] I have read the [CONTRIBUTING](https://github.com/litmuschaos/litmus/blob/master/CONTRIBUTING.md) doc
- [ ] I have signed the commit for DCO to be passed.
- [ ] Lint and unit tests pass locally with my changes
- [ ] I have added tests that prove my fix is effective or that my feature works (if appropriate)
- [ ] I have added necessary documentation (if appropriate)

## Dependency
- Please add the links to the dependent PR need to be merged before this (if any).

### CODE REVIEW

**Review by ishangupta-ds — APPROVED** [review]
LGTM

**Review by ishangupta-ds — COMMENTED** [review]
@rajdas98 can we also add this change for Okteto dev manifest

**Comment by rajdas98** [comment]
> @rajdas98 can we also add this change for Okteto dev manifest

Thanks. Done

File: litmus-portal/README.md [hunk 1]
```diff
@@ -39,7 +39,6 @@ kubectl apply -f https://raw.githubusercontent.com/litmuschaos/litmus/master/lit
 curl https://raw.githubusercontent.com/litmuschaos/litmus/master/litmus-portal/namespaced-K8s-template.yml --output litmus-portal-namespaced-K8s-template.yml
 envsubst < litmus-portal-namespaced-K8s-template.yml > ${LITMUS_PORTAL_NAMESPACE}-ns-scoped-litmus-portal-manifest.yml
 kubectl apply -f ${LITMUS_PORTAL_NAMESPACE}-ns-scoped-litmus-portal-manifest.yml -n ${LITMUS_PORTAL_NAMESPACE}
-kubectl apply -f https://raw.githubusercontent.com/litmuschaos/litmus/master/litmus-portal/platforms/okteto/hello-world-AUT.yml -n ${LITMUS_PORTAL_NAMESPACE}
```
Comment (ishangupta-ds) [inline litmus-portal/README.md:42]: This might be required @rajdas98 for running the chaos workflow specifically meant for Okteto, we have it on portal.

**Review by ishangupta-ds — CHANGES_REQUESTED** [review]

File: litmus-portal/README.md (hunk 1 above)
Comment (rajdas98) [reply litmus-portal/README.md:42]: can we move this to oketeto readme? These installation steps need to be generic for everyone

Comment (ishangupta-ds) [reply litmus-portal/README.md:42]: But for namespaces scope unless someone installs this hello world app, no other predefined workflow will work @rajdas98 cc: @ksatchit

**Review by ishangupta-ds — APPROVED** [review]

**Pull request merged**